Update C# Rider install steps for now built-in plugin "Godot Support" (#10323)
godotengine/godot-docs: tutorials/scripting/c_sharp/c_sharp_basics.rst

In Rider:

- Set **MSBuild version** to **.NET Core**.

> - Install the **Godot support** plugin (Not needed for Rider 2024.2 and above).

Reviewer: Possibly:
```suggestion
- If you are using a Rider version below 2024.2, install the **Godot support** plugin.
```
To treat newer versions as the norm. We might not update this page for a while and this is slightly more future-proof.

Or:
```suggestion
- Install the **Godot support** plugin (Not needed for Rider 2024.2 and above, since it is built-in now).
```
Explain the reasoning a bit more explicitly.

Or:
```suggestion
- If you are using a Rider version below 2024.2, install the **Godot support** plugin. This functionality is now built into Rider.
```
As a combination of both.
